CHANGELOG.md

```diff
@@ -14,7 +15,7 @@
   - [#114](https://github.com/Datatamer/unify-client-python/issues/114) Add support for generating pairs estimate
   - [#106](https://github.com/Datatamer/unify-client-python/issues/106) Add support for initializing a source dataset
   - [#107](https://github.com/Datatamer/unify-client-python/issues/107) Add support for creating a dataset attribute
-  
```

Comment:
This extraneous whitespace change is still here!

Reply:
what do you mean by this?

Reply:
@CaspianA1 I think what @nbateshaus means here is that this change (though correct) is irrelevant to the feature/bug you are working on.

We *should* fix this, but @nbateshaus thinks we should do so as its own commit within this PR (not as part of the commit where you actually address the bug)